astronomer/providers/amazon/aws/example_dags/example_emr_eks_containers_job.py

```diff
@@ -122,15 +99,15 @@ def create_emr_virtual_cluster_func(task_instance: Any) -> None:
         f"sh $AIRFLOW_HOME/dags/example_create_eks_kube_namespace_with_role.sh ",
     )
 
-    # Task to create EMR virtual cluster
-    create_emr_virtual_cluster = PythonOperator(
-        task_id="create_emr_virtual_cluster",
-        python_callable=create_emr_virtual_cluster_func,
+    create_emr_virtual_cluster = EmrEksCreateClusterOperator(
+        task_id="create_emr_eks_cluster",
+        virtual_cluster_name=VIRTUAL_CLUSTER_NAME,
+        eks_cluster_name=EKS_CLUSTER_NAME,
+        eks_namespace=EKS_NAMESPACE,
     )
 
-    VIRTUAL_CLUSTER_ID = (
-        "{{ task_instance.xcom_pull(task_ids='create_emr_virtual_cluster', " "key='virtual_cluster_id') }}"
-    )
+    VIRTUAL_CLUSTER_ID = str(create_emr_virtual_cluster.output)
```

Comment:
I dont think we need that explicit `str()` conversion, do we?
```suggestion
    VIRTUAL_CLUSTER_ID = create_emr_virtual_cluster.output
```